frontend/public/components/_resource-dropdown.scss

```diff
@@ -1,6 +1,14 @@
 .co-type-selector {
   margin-right: -2px;
   min-width: 0;
+  .btn-dropdown__content-wrap {
+    .show {
+      &::before {
+        content: " - "
+      }
+      display: inline !important;
```

Comment:
Nit:  I'm inclined to put the declarations that belong to the rule first so it's obvious what rule they belong to.  With SASS nesting it can get hard to know what declarations belong to what.